### DISCUSSION TITLE
holding the back button should hold and not release

### DISCUSSION THREAD

**Discussion by joeblack2k** [LizardByte/community#583] [Sunshine]
when you enable, holding the back button presses the guide button it would be nice to have the option that it holds the guide button instead of pressing it so steam shortcuts can be made.. so when set to 2000 then after 2 seconds the guide button is hold and you can press A to launch the quick settings or release the button to launch the guide menu

[a comment by an automated account is not shown]

**Comment by joeblack2k** [comment]
its still a good idea though..

**Reply by ReenigneArcher** [reply]
we accept PRs

**Reply by joeblack2k** [reply]
what is a PR?

**Reply by ReenigneArcher** [reply]
pull request

**Reply by joeblack2k** [reply]
oke, i will ask claude code to fix it
